libs/category/driver/magento/src/transformers/category-response-transform.service.spec.ts

```diff
@@ -44,13 +44,15 @@ describe('DaffMagentoCategoryResponseTransformService', () => {
   let magentoCategoryTransformerServiceSpy: jasmine.SpyObj<DaffMagentoCategoryTransformerService>;
   let magentoCategoryPageConfigurationTransformerServiceSpy: jasmine.SpyObj<DaffMagentoCategoryPageConfigTransformerService>;
   const stubMediaUrl = 'mediaUrl';
+  let magentoProductsTransformService: DaffMagentoProductsTransformer;
 
   beforeEach(() => {
     magentoCategoryTransformerServiceSpy = jasmine.createSpyObj('DaffMagentoCategoryTransformerService', ['transform']);
     magentoCategoryPageConfigurationTransformerServiceSpy = jasmine.createSpyObj('DaffMagentoCategoryPageConfigTransformerService', ['transform']);
 
     TestBed.configureTestingModule({
       providers: [
+        DaffMagentoProductsTransformer,
```

Comment:
isn't this provided in root?

same for the rest of transformer services/tests

Reply:
true. Fixed